frontend/src/app/(auth)/login/page.tsx

```diff
@@ -1,38 +1,31 @@
 "use client";
 
-import {
-    FacebookAuthProvider,
-    getAuth,
-    GoogleAuthProvider,
-    signInWithCustomToken,
-    signInWithPopup
-} from "firebase/auth";
 import Link from "next/link";
-import { toast } from "sonner";
 import { useForm } from "react-hook-form";
 import { FcGoogle } from "react-icons/fc";
+import { FaGithub } from "react-icons/fa";
 import { useRouter } from "next/navigation";
-import { AxiosError } from "axios";
-import { BsFacebook } from "react-icons/bs";
 import { Eye, EyeOff, RefreshCcw } from "lucide-react";
 import React, { Fragment, useCallback, useState } from "react";
+import { getAuth, GithubAuthProvider, GoogleAuthProvider, signInWithCustomToken, signInWithPopup } from "firebase/auth";
 
+import { cn } from "@/lib/utils";
```

Comment:
Would recommend using a descriptive variable name like the same you have down below and in other places